tests/app/helpers/test_template_helpers.py

```diff
@@ -17,49 +17,54 @@
     WelshCensusSurveyConfig,
 )
 from app.survey_config.survey_type import SurveyType
+from tests.app.helpers.conftest import (
+    expected_footer_business_theme,
+    expected_footer_business_theme_no_cookie,
+    expected_footer_census_theme,
+    expected_footer_census_theme_no_cookie,
+    expected_footer_census_welsh_theme,
+    expected_footer_nisra_theme,
+    expected_footer_social_theme,
+    expected_footer_social_theme_no_cookie,
+)
 
 DEFAULT_URL = "http://localhost"
 
 
-def test_footer_context_census_theme(app: Flask, expected_footer_census_theme):
+@pytest.mark.parametrize(
+    "expected_footer, theme, survey_config",
```

Comment:
(Non-blocker) Minor, it would be better for `expected_footer` to be the last arg, like in most other cases. So it is more like `GIVEN a survey WHEN I have theme and suvery_config THEN I get the expected footer.

Reply:
Ok, added, thanks!